ui2/package.json

```diff
@@ -18,6 +18,7 @@
     "d3": "^6.5.0",
     "date-fns": "^2.17.0",
     "jsonwebtoken": "^8.5.1",
+    "luxon": "^1.26.0",
```

Comment:
Think I made same comment last time: do we have some policy for inclusion of libraries? At this rate we're adding a couple every week ... ?

Reply:
I've replaced `moment` with `luxon` as `moment` is considered legacy with `luxon` as its replacement. 

As for the policy for inclusion, I haven't seen anything regarding policy but I would expect it should be passing a Blackduck scan to check for vulnerabilities.